src/apprt/gtk/wayland.zig

```diff
@@ -0,0 +1,125 @@
+const std = @import("std");
+const c = @import("c.zig").c;
+const wayland = @import("wayland");
+const wl = wayland.client.wl;
+const org = wayland.client.org;
+const build_options = @import("build_options");
+
+const log = std.log.scoped(.gtk_wayland);
+
+/// Wayland state that contains application-wide Wayland objects (e.g. wl_display).
+pub const AppState = struct {
+    display: *wl.Display,
+    blur_manager: ?*org.KdeKwinBlurManager = null,
+
+    pub fn create(display: ?*c.GdkDisplay) ?AppState {
```

Comment:
The `create` function idiomatically in Zig allocates and returns a pointer. This should be `init`.